registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/eventbus/KafkaMosipEventBus.java

```diff
@@ -446,46 +444,27 @@ public void consumerHealthCheck(EventHandler<EventDTO, Handler<AsyncResult<Healt
 				healthCheckDTO.setEventBusConnected(false);
 				healthCheckDTO.setFailureReason(f.cause().getMessage());
 			}
-			try {
-				eventDTO.setBody(new JsonObject(JsonUtil.objectMapperObjectToJson(healthCheckDTO)));
-			} catch (IOException e) {
-				healthCheckDTO.setEventBusConnected(false);
-				healthCheckDTO.setFailureReason(e.getMessage());
-				eventHandler.handle(eventDTO, res -> {
-				});
-			}
-			eventHandler.handle(eventDTO, res -> {
-
-			});
+			eventHandler.handle(healthCheckDTO);
 		});
 
 	}
 
 	@Override
-	public void sendHealthCheck(EventHandler<EventDTO, Handler<AsyncResult<HealthCheckDTO>>> eventHandler,
+	public void senderHealthCheck(Handler<HealthCheckDTO> eventHandler,
 			String address) {
-		EventDTO eventDTO = new EventDTO();
-		HealthCheckDTO healthCheckDTO = new HealthCheckDTO();
-		kafkaConsumer.listTopics(f -> {
-			if (f.succeeded()) {
-				healthCheckDTO.setEventBusConnected(true);
 
-			} else {
+		HealthCheckDTO healthCheckDTO = new HealthCheckDTO();
+		KafkaProducerRecord<String, String> producerRecord = KafkaProducerRecord.create(address,
+				HealthConstant.PING, HealthConstant.PING);
+		kafkaProducer.write(producerRecord, handler -> {

```

Comment:
This will not create a message into our outbus with content as ping, camel cannot understand this message

Reply:
Removed the code